lib/ip_wrangler/main.rb

```diff
@@ -1,26 +1,22 @@
-$config = YAML.load_file('config.yml')
+$config = YAML.load_file('./config.yml')
 
 use Rack::Auth::Basic, 'Restricted Area' do |username, password|
-  [username, password] == [$config[:username], $config[:password]]
+  [username, password] == [$config['username'], $config['password']]
 end
 
-$logger = Logger.new($config[:log_file_path])
+$logger = Logger.new(STDOUT)
 
-$nat = NAT.new($config, $config[:db_name], $config[:iptables_chain_name], $logger)
+$nat = IpWrangler::NAT.new($config, './' + $config['db_name'], $config['iptables_chain_name'], $logger)
 
 def sandbox(&block)
   begin
-    content_type 'application/json'
+    content_type('application/json')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.